ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Analyzers/RedundantAttributeSuffixAnalyzer.fs

```diff
@@ -11,8 +11,10 @@ type RedundantAttributeSuffixAnalyzer() =
     inherit ElementProblemAnalyzer<IAttribute>()
 
     override x.Run(attribute, _, consumer) =
+        if isNull attribute.ReferenceName then () else
+
         let attributeName = attribute.ReferenceName.ShortName
-        if not (attributeName.EndsWith("Attribute")) then () else
+        if attributeName = "Attribute" || not (attributeName.EndsWith("Attribute")) then () else
```

Comment:
Just a small possible improvement: in the most cases we're dealing with names that are not `"Attribute"` so let's check this name only as a last check.
```suggestion
        if not (attributeName.EndsWith("Attribute") && attirbuteName.Length > "Attribute".Length) then () else
```